### PR TITLE
Fb/cart fragment

### PR DESCRIPTION
**Pull request by surajprakashpatel**

### CODE REVIEW

File: app/src/main/java/com/zariya/zariya/payment/presentation/fragment/CheckoutFragment.kt [hunk 1]
```diff
@@ -22,6 +23,16 @@ class CheckoutFragment : BaseFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        activity?.setActionBar(binding.toolbar)
+        activity?.actionBar?.setDisplayHomeAsUpEnabled(true)
+        activity?.actionBar?.setDisplayShowHomeEnabled(true)
+        activity?.actionBar?.title = "Almost there!"
```
Comment (VivekSharma811) [inline app/src/main/java/com/zariya/zariya/payment/presentation/fragment/CheckoutFragment.kt:29]: Move to Strings xml

File: app/src/main/java/com/zariya/zariya/services/presentation/adpater/ServiceAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.zariya.zariya.services.presentation.adpater
+
+class ServiceAdapter {
```
Comment (VivekSharma811) [inline app/src/main/java/com/zariya/zariya/services/presentation/adpater/ServiceAdapter.kt:3]: Remove the class if not needed

File: app/src/main/java/com/zariya/zariya/services/presentation/fragment/MyServicesFragment.kt [hunk 3]
```diff
@@ -5,17 +5,32 @@ import androidx.fragment.app.Fragment
...
         return binding.root
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        activity?.setActionBar(binding.toolbar)
+        activity?.actionBar?.setDisplayHomeAsUpEnabled(true)
+        activity?.actionBar?.setDisplayShowHomeEnabled(true)
+        activity?.actionBar?.title = "My Cart"
```
Comment (VivekSharma811) [inline app/src/main/java/com/zariya/zariya/services/presentation/fragment/MyServicesFragment.kt:30]: Move to strings xml

File: app/src/main/res/layout/fragment_my_services.xml [hunk 4]
```diff
@@ -7,14 +7,73 @@
...
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toBottomOf="parent"
-            android:text="Messages" />
+            app:navigationIcon="@drawable/ic_back_arrow"
+            android:visibility="gone"/>
```
Comment (VivekSharma811) [inline app/src/main/res/layout/fragment_my_services.xml:18]: Why is toolbar used if visibility is gone?

File: app/src/main/res/layout/fragment_my_services.xml [hunk 5]
```diff
@@ -7,14 +7,73 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/toolbar"
+            android:visibility="visible"/>
+
 
+        <LinearLayout
+            android:id="@+id/empty_cart"
```
Comment (VivekSharma811) [inline app/src/main/res/layout/fragment_my_services.xml:31]: Follow only camelCasing for naming variables and ids

File: app/src/main/res/layout/fragment_my_services.xml [hunk 6]
```diff
@@ -7,14 +7,73 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:elevation="1dp"
+            android:gravity="center"
+            android:orientation="vertical"
+            android:visibility="visible"
+            app:layout_constraintTop_toTopOf="@+id/toolbar">
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="150dp"
```
Comment (VivekSharma811) [inline app/src/main/res/layout/fragment_my_services.xml:41]: Use Sdp instead of dp

**Review by VivekSharma811 — APPROVED** [review]

**Review by Mahima081 — APPROVED** [review]

**Pull request merged**